game-core/src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -209,7 +209,7 @@ public void loadDefaultGameNewThread() {
    * on startup.
    */
   public void loadDefaultGameSameThread() {
-    final String userPreferredDefaultGameUri = ClientSetting.defaultGameUri.value();
+    final String userPreferredDefaultGameUri = ClientSetting.defaultGameUri.getValue().orElse("");
```

Comment:
The code below already handles an empty string as a signal that the setting is not defined.